packages/shared/src/components/squads/Members/PrivilegedMemberItem.tsx

```diff
@@ -17,14 +18,16 @@ export function PrivilegedMemberItem({
     <ProfileTooltip userId={user.id} tooltip={{ placement: 'bottom' }}>
       <ProfileLink
         href={user.permalink}
-        className="flex flex-row items-center rounded-10 border border-border-subtlest-tertiary p-2"
+        className="flex flex-row items-center gap-2.5 rounded-10 border border-border-subtlest-tertiary p-2"
```

Comment:
We don't normally use `.5` values. Probably an inaccuracy from the Figma board, feel free to confirm with the design team.
```suggestion
        className="flex flex-row items-center gap-2 rounded-10 border border-border-subtlest-tertiary p-2"
```